integration-test-groups/aws2/aws2-cw/pom.xml

```diff
@@ -39,6 +39,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jackson</artifactId>
```

Comment:
@ppalaga If I remember well, we try our best not to use `quarkus-resteasy-jackson` in tests because it registers some native reflections that could potentially hide bug in our extensions ? If yes, what could we advise in the test at hand ?

Reply:
@aldettinger  I can refactor the test to use a "custom" serialization. I tried that in this case, but it seems like a potential cause of errors. Maybe we could add some kind of "util", "helper" class to serialize and deserialize maps into some test extension, what do you think?

Reply:
Yes, we should come with an idea that improve the situation, if feasible. On top of my mind, I don't have much. Would passing a Map<String, String> works in plain text ? Maybe we could then add long/int as extra parameters... Not sure it would make it.

Reply:
fixed